pendant/readme.txt

```diff
@@ -42,3 +42,39 @@ This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 GNU General Public License for more details.
+
+This theme bundles the following third-party resources:
+
+MET Museum
+License : Public Domain
+Source  : https://www.metmuseum.org/art/collection/search/243039
+          https://www.metmuseum.org/art/collection/search/248813
+          https://www.metmuseum.org/art/collection/search/452273
+	  https://www.metmuseum.org/art/collection/search/256970
+	  https://www.metmuseum.org/art/collection/search/236772
+	  https://www.metmuseum.org/art/collection/search/242952
+
+Smithsonian
+License : Creative Commons CC0
+Source  : https://www.si.edu/object/wristwatch-hour-angle-lindbergh-king-collection:nasm_A20040287020
+
+PxHere
+License : Creative Commons CC0
+Source  : https://pxhere.com/en/photo/775017
+          https://pxhere.com/pt/photo/41666
```

Comment:
I don't see these two images being used in the theme.

Reply:
We only need to credit the images we are including in the assets files, the rest can be used in dotcom as a headstart content without the need for credit